Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM dmstr/php-yii2:7.0-fpm-1.9-beta2-alpine-nginx
```

Comment:
I was thinking about moving the Dockerfile in the build script, e.g. `cp tests/docker/Dockerfile-mysql Dockerfile` and then run docker-compose, so that they are not located in the repo root but are moved to repo root when needed.